constants/battle_constants.asm

```diff
@@ -311,6 +311,10 @@ QUICK_PACK   EQU 1 << QUICK_PACK_F
 	const LOSE
 	const DRAW
 
+DEF BATTLERESULT_CAUGHT_POKEMON EQU 6
```

Comment:
Generally bitflag constants end in `_F`.

Reply:
Also `const_def` would be used even for such a short sequence:
```
const_def 6
const BATTLERESULT_CAUGHT_POKEMON  ; 6
const BATTLERESULT_BOX_FULL        ; 7
```

Reply:
This section was actually just lifted from pokecrystal.. but easy enough.